src/System.CommandLine.Tests/Help/Approvals/HelpBuilderTests.Help_describes_default_values_for_complex_root_command_scenario.approved.txt

```diff
@@ -1,24 +1,24 @@
-﻿the-root-command:
+﻿the-root-command
   Test description
 
 Usage:
-  the-root-command [options] <the-root-arg-no-description-no-default> [<the-root-arg-no-description-default> <the-root-arg-no-default> [<the-root-arg> [<the-root-arg-enum-default>]]]
+  the-root-command <the-root-arg-no-description-no-default> [<the-root-arg-no-description-default> <the-root-arg-no-default> [<the-root-arg> [<the-root-arg-enum-default>]]] [options]
```

Comment:
In most CLI synopses, options precede arguments. In many cases the options are actually fully expanded. Something to consider?

Reply:
That is a good point, I will fix it.